nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/main/java/org/apache/nifi/util/db/JdbcProperties.java

```diff
@@ -78,4 +77,15 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .required(true)
             .build();
+
+    // Registry-only versions of Default Precision and Default Scale properties
+    public static final PropertyDescriptor REGISTRY_ONLY_DEFAULT_PRECISION =
```

Comment:
I'm not sure that I understand what "registry" means in this context

Reply:
I'll change them to say `VARIABLE_REGISTRY_ONLY`, hopefully that makes it clearer that it is like the base property but doesn't support FlowFile attributes

Reply:
Ah, gotcha. Thanks.